src/index.html

```diff
@@ -13,6 +13,40 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <section class="section">
+      <ul class="calendar calendar--start-day-sun calendar--month-length-31">
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
+        <li class="calendar__day"></li>
```

Comment:
If you're manually creating each day of the calendar with `li`, this is not scalable or maintainable. Consider generating these elements dynamically with a script, or if they must be in HTML, use a templating engine to avoid repetition.